### DISCUSSION TITLE
add the ability to be a portable application

### DISCUSSION THREAD

**Discussion by bassuny3003** [valinet/ExplorerPatcher#2299] [Ideas]
It is better to work as a portable application.

so can you add way to open the GUI without installing the tool at all

**Comment by pyrates999** [comment]
You can manually extract it to then install it: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

Do you mean to install it to one user only and not any other user?

**Reply by bassuny3003** [reply]
when i try what in the link you give to me it just extracts the dll files in the folder i choose but it has no exe file to run the GUI of the toll itself 

i make this code in the short cut to run the app form other path 
"C:\Windows\System32\cmd.exe /c start rundll32.exe "**%CD%**\ExplorerPatcher.amd64.dll",ZZGUI"

but i work only when i still installing the tool. 

after using "**ep_setup.exe /uninstall**"

this shortcut path work to only run the GUI but has no effect to the start menu like "Hide recommended section."


SO 

how to run the tool without installing it "as portable app" to the current user or to any user doesn't matter. 

thanks, and sorry for my bad English.

**Reply by pyrates999** [reply]
It currently doesn't support being ran as a portable application.
